### DISCUSSION TITLE
Log used prompt on generations when using Langchain

### DISCUSSION THREAD

**Discussion by TobinShaw** [langfuse/langfuse#1309] [Ideas]
I want to use the integrated version of Langchain and check the prompt version on the dashboard. How can I do this? It seems like I can only add trace and span  in the callbackhandler, but I can't add a generation and pass the prompt as a parameter.

**Comment by marcklingen** [comment]
Hi Tobin, currently that's unsupported. I've just rephrased it a bit and added this to the ideas list.
It could be interesting to parse prompt templates and detect if they contain a prompt managed in Langfuse.

**Comment by hassiebp** [comment]
Linking Langfuse prompts to Langchain executions has been one of the most requested features recently, and we have just released support for it 🫡 🥳

For more details, please check out:

- [Docs on how to integrate, see tabs for Langchain in examples](https://langfuse.com/docs/prompts/get-started#link-with-langfuse-tracing-optional)
- [Python SDK release](https://github.com/langfuse/langfuse-python/releases/tag/v2.49.0)
- [JS SDK release](https://github.com/langfuse/langfuse-js/releases/tag/v3.25.0)
